modules/check_in/config/initializers/statsd.rb

```diff
@@ -53,17 +58,15 @@
 
   private
 
-  def check_in_type(params)
-    check_in_param = params[:checkInType]
-    check_in_param = params.dig(:session, :check_in_type) if check_in_param.nil?
-    check_in_param == 'preCheckIn' ? 'pre_check_in' : 'check_in'
-  end
-
   def metric_prefix(headers, params)
-    check_in_param = params[:checkInType] || params.dig(:session, :check_in_type)
+    if params[:facility_type]&.downcase == 'oh'
```

Comment:
I was working on the same part of code for SessionsController and had this instead:

```
    check_in_param = params[:checkInType] || params.dig(:session, :check_in_type)
    facility_type_param = params[:facilityType] || params.dig(:session, :facility_type)

    prefix = check_in_param == 'preCheckIn' ? 'pre_check_in' : 'check_in'
    prefix = facility_type_param unless facility_type_param.nil?
    prefix += '.synthetic' if headers.key?('Sec-Datadog')
    prefix
```